src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -440,45 +462,36 @@ JSON example:
 ```JSON
 {
   "PhaseEncodingDirection": "j-",
-  "TotalReadoutTime": 0.095
+  "TotalReadoutTime": 0.095,
+  "B0FieldSource": ["phasediff_fmap0", "pepolar_fmap0"]
 }
 ```
 
 ## Fieldmap data
```

Comment:
I have refactored this section, as it clearly had different formatting. Now it should be more consistent with the above, and the coexistence of the new `B0FieldIdentifier` and `IntendedFor` should not flow unnaturally.

In terms of the spec, this is a refactor (i.e., no semantics about how to encode fieldmaps unrelated to `B0FieldIdentifier` should be found)